.travis.yml

```diff
@@ -1,15 +0,0 @@
-language: node_js
-node_js:
- - "node"
-addons:
-  apt:
-    packages:
-      # Ubuntu 16+ does not install this dependency by default, so we need to install it ourselves
-      - libgconf-2-4
-install:
-  - npm ci
-  - npm install -g codecov
-script:
-  - npm run test:ci
-after_success:
-  - npm run test:unit:coverage:ci
```

Comment:
Just a passing thought, but I wonder if we should add a bunch of these to docs for those wanting to integrate with various CIs 🤔

Reply:
That's a really good idea!

Reply:
Cookbook for each CI we encounter 👍